data-plane/examples/Taskfile.yaml

```diff
@@ -32,3 +32,27 @@ tasks:
     dir: src/sdk-mock
     cmds:
       - cargo run -- --config ../../../config/base/client-config.yaml --local-agent client --remote-agent server --message "hey there!"
+
+  docker:build-all:
+    desc: "Build all Docker images"
+    dir: "../.."
+    cmds:
+      - docker build -t slim-server -f data-plane/examples/src/docker/Dockerfile.slim .
+      - docker build -t mock-agent-server -f data-plane/examples/src/docker/Dockerfile.mock-agent-server .
+      - docker build -t mock-agent-client -f data-plane/examples/src/docker/Dockerfile.mock-agent-client .
```

Comment:
Update here accordingly using the `--target` argument

Reply:
done